src/ir_Hitachi.cpp

```diff
@@ -654,6 +654,21 @@ void IRHitachiAc424::setFan(const uint8_t speed) {
   }
 }
 
+uint8_t IRHitachiAc424::getButton(void) {
+  return remote_state[kHitachiAc424ButtonByte];
+}
+
+// The remote sends the type of button pressed on send
+void IRHitachiAc424::setButton(const uint8_t type) {
+  remote_state[kHitachiAc424ButtonByte] = type;
```

Comment:
Very minor Style Nitpick: Please use a different variable name rather than `type`.
value, command, whatever. It's just close to a reserved word. e.g. `typedef`. It is used in other languages.  It is just making my skin crawl. Technically nothing wrong .. but .. shudder. ;-)

Reply:
changed